app/app.go

```diff
@@ -375,6 +375,10 @@ func New(
 			TxPriority:      mempool.NewDefaultTxPriority(),
 			SignerExtractor: evmapp.NewEthSignerExtractionAdapter(mempool.NewDefaultSignerExtractionAdapter()),
 			MaxTx:           maxTxs,
+			TxReplacement: func(op, np int64, oTx, nTx sdk.Tx) bool {
+				// tx is only replaced if new priority is higher than old priority
+				return np > op
```

Comment:
do we need align how [ethereum](https://github.com/ethereum/go-ethereum/blob/release/1.11/core/txpool/list.go#L284) replace tx

Reply:
1. priceBump parameter
2. Require both FeeCap and TipCap larger than the old one.

Reply:
the same nonce tx will fail in checkTx in the first place, might not be trivial to support this feature.